etc/kayobe/kolla/config/bifrost/bifrost.yml

```diff
@@ -8,7 +8,7 @@ use_cirros: true
 {% if os_distribution == 'centos' %}
 cirros_deploy_image_upstream_url: "https://cloud.centos.org/centos/9-stream/x86_64/images/CentOS-Stream-GenericCloud-9-20221206.0.x86_64.qcow2"
 {% elif os_distribution == 'rocky' %}
-cirros_deploy_image_upstream_url: "https://dl.rockylinux.org/pub/rocky/9/images/x86_64/Rocky-9-GenericCloud.latest.x86_64.qcow2"
+cirros_deploy_image_upstream_url: "https://dl.rockylinux.org/vault/rocky/9.3/images/x86_64/Rocky-9-GenericCloud.latest.x86_64.qcow2"
```

Comment:
Should we add a comment with a link to a bug raised in RL bug tracker?

Reply:
Do you think this is the same bug? https://bugs.rockylinux.org/view.php?id=6832

Reply:
Yes, that's the one I mentioned to Rocky Linux devs and they said it should be fixed with latest images - I wrote to them today it's not fixed.